base.cfg

```diff
@@ -0,0 +1,122 @@
+[buildout]
+versions = versions
+parts =
+    instance
+    compileresources
+    ipython
+    i18ndude
+    omelette
+    robot
+    test
+    coverage-test
+    coverage-report
+
+include-site-packages = false
+
+find-links +=
+    http://dist.repoze.org/
+    http://effbot.org/downloads/
+
+[versions]
+zc.buildout =
+setuptools = 19.0
+# plone.app.workflow = 2.1.9
+# XXX: use a newer version to make the ordered select input widget work:
+# https://github.com/bluedynamics/bda.plone.shop/issues/55
+# Remove when z3c.form is updated in Plone.
+# z3c.form = 3.2.6
+
+# robot
+#plone.app.robotframework = 0.9.1
+#robotframework = 2.8.4
+#robotframework-selenium2library = 1.5.0
+#robotsuite = 1.4.3
+#selenium = 2.39.0
+#robotframework-selenium2screenshots = 0.4.0
+#sphinxcontrib-robotframework = 0.4.3
+#robotframework-debuglibrary = 0.3
```

Comment:
what should happen with all this unused version pinnings? if we provide, we should split up p4 and p5, or otherwise remove entirely